src/cli/spinner.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @fileoverview CLI Spinner wrapper.
+ *
+ */
+
+'use strict';
+
+const ora = require('ora');
+const logSymbols = require('log-symbols');
+
+/**
+ * CLI Spinner
+ *
+ * @class Spinner
+ */
+class Spinner {
+  /**
+   * Creates an instance of Spinner.
+   * @param {Boolean} enable
+   * @memberof Spinner
+   */
+  constructor(enable) {
+    this.enable = enable;
```

Comment:
not sure we need this class variable since we can just check the existence of this.spinner. Also, not really on board with the idea of have a enable flag which just disables all functionality of the class, wouldn't we just not create the spinner class? I guess this way says use from have to check in cli.js if it's enabled or not.